docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -52,13 +47,13 @@ SOURCE_GALERA_MEMBERS=(
 SOURCE_DB_ROOT_PASSWORD=$(cat ~/tripleo-standalone-passwords.yaml | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
 ----
 
-* Prepare MariaDB copy directory and the adoption helper pod
+* Prepare the MariaDB copy directory and the adoption helper pod:
 
-* Create a temporary folder to store adoption helper pod (pick storage requests to fit MySQL database size):
+. Create a temporary folder to store the adoption helper pod. Choose storage requests that fit the MySQL database size:
```

Comment:
Does the customer need to do something? E.g, enter a value in the “requests:” parameter?

Reply:
Sorry i skipped this earlier. The folder will be removed (it is unused), there is a bug for it, i'll post a patch.

The sentence about storage requests refers to this part of the snippet below:

```
  resources:
    requests:
      storage: 10Gi
```

The storage request needs to be big enough to fit the overcloud databases stored in MariaDB. In the example we request 10 gigabytes.